web-components/src/components/organisms/ProjectFilters/ProjectFilters.tsx

```diff
@@ -111,13 +111,11 @@ export default function ProjectFilters({
                 disableRipple
                 onClick={() => handleExpand(index)}
                 sx={{ fontWeight: '700' }}
-                className="text-sc-text-link"
+                className="text-sc-text-link uppercase"
               >
                 {isExpanded[index] ? labels.collapse : labels.expand}
               </ButtonBase>
             )}
-
-            <Divider sx={{ my: 5 }} />
```

Comment:
it should only be removed for the last filter section, not for all see design https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=1204-174906&node-type=instance&m=dev